templates/generic/rpi4_reference_dunfell.json

```diff
@@ -94,12 +94,27 @@
                     "rw",
                     "X-dobby.optional"
                 ]
+            },
+            {
+            "source": "/tmp",
+            "destination": "/tmp",
+            "options": [
+                "rbind",
+                "nosuid",
+                "nodev",
+                "rw"
+            ],
+            "type": "bind"
             }
         ],
         "envvar": [
             "LD_PRELOAD=/usr/lib/libGLESv2.so.2:/usr/lib/libwayland-egl.so.1",
             "XDG_RUNTIME_DIR=/tmp",
-            "RIALTO_SOCKET_PATH=/tmp/rialto-0"
```

Comment:
This must stay /tmp/rialto-0 if you mount bind the rialto socket as I mentioned above.